examples/other/integration_pytorch.py

```diff
@@ -0,0 +1,530 @@
+# run_example = false
+
+"""
+Pytorch Integration
+=============================
+
+Pygfx Integration with PyTorch Lightning and PyTorch Geometric: A Practical Example.
+
+In this demonstration, we train a graph neural network on a 3D mesh. As the training progresses, we simultaneously render
+the results on a pygfx window in a separate process. The network aims to overfit the Gaussian curvature of each
+mesh point, striving to match the known curvature values precisely.
+
+"""
+
+# standard library
+from pathlib import Path
+from typing import (
+    Optional,
+    Tuple,
+    List,
+    Any,
+    cast)
+from dataclasses import dataclass
+import queue
+
+# numpy
+import numpy as np
+
+# sklearn
+from sklearn.decomposition import PCA
+
+# trimesh
+import trimesh
+
+# libigl
+import igl
+
+# pygfx
+import pygfx as gfx
+from pygfx import (
+    Viewport,
+    Renderer,
+    Scene,
+    Group,
+    Background,
+    BackgroundMaterial,
+    Color,
+    Camera,
+    PerspectiveCamera,
+    NDCCamera,
+    PointLight,
+    OrbitController,
+    Geometry,
+    WorldObject,
+    Mesh,
+    MeshPhongMaterial,
+    Buffer)
```

Comment:
I think it would be clearer to just use the `gfx` namespace imported above rather than import every class that is used.

Reply:
Thanks for the review, I will address it in the next commit.

Reply:
Fixed